evadb/parser/evadb.lark

```diff
@@ -20,12 +20,16 @@ context_statement: use_statement
 
 create_database: CREATE DATABASE if_not_exists? uid create_database_engine_clause
 
+create_application: CREATE APPLICATION if_not_exists? uid create_application_engine_clause
+
 colon_param: string_literal ":" string_literal
 
 colon_param_dict: LR_CURLY_BRACKET colon_param ("," colon_param)* RR_CURLY_BRACKET
 
 create_database_engine_clause: WITH ENGINE "=" string_literal "," PARAMETERS "=" colon_param_dict 
 
+create_application_engine_clause: WITH ENGINE "=" string_literal "," PARAMETERS "=" colon_param_dict
```

Comment:
I think we can consolidate `create_database_engine_clause` and `create_application_engine_clause`. They are essentially the same.